fdbclient/StorageServerInterface.h

```diff
@@ -92,6 +113,8 @@ struct StorageServerInterface {
 	PublicRequestStream<struct FetchCheckpointRequest> fetchCheckpoint;
 	PublicRequestStream<struct FetchCheckpointKeyValuesRequest> fetchCheckpointKeyValues;
 
+	RequestStream<struct UpdateCommitCostRequest> updateCommitCostRequest;
```

Comment:
This is protocol incompatible with 7.1. Is there another way to fix the issue for 7.1?